BootROMs/src/cgb_boot.asm

```diff
@@ -198,15 +191,23 @@ ELSE
     call PlaySound
 ENDC
     call Preboot
-
-; Will be filled with NOPs
+    ld a, [rBANK] ; Special MiSTer read register to give menu boot options
+    bit 0, a
+    jr z, .cgbmode
```

Comment:
```suggestion
    cp $42
    jr nz, .cgbmode
```

Reply:
I did read through your suggestion from your earlier comment, I was going to write a comment later explaining these changes and a few others (I am still working on a couple of things right now). Wasn't expecting a review to come in!

I had a think about this, and so long as the $FF50 register is overriden while `boot_rom_enabled` is active, we have complete control over the read/write accesses. cgb_boot.asm does not read from $FF50, except here. So really, we can use a single bit rather than an 8 bit literal. I don't see the benefit of comparing to $42. 

Does the real cgb bootrom read from $FF50, and is that a cause for concern?

Reply:
Yes you are right. It doesn't really matter since the boot roms don't normally read it anyway.